documentation/modules/managing/proc-cluster-recovery-volume.adoc

```diff
@@ -0,0 +1,232 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="cluster-recovery-volume_{context}"]
+= Recovering a cluster from persistent volumes
+
+This procedure describes how to recover a deleted cluster from persistent volumes (PVs).
+
+When you get to the step to recreate your cluster, you have two options:
+
+. Use _Option 1_ when you can recover all `KafkaTopic` resources.
++
+In this situation, the Topic Operator identifies that topics exist in Kafka, but the `KafkaTopic` resources do not exist.
```

Comment:
The _In this situation_ sounds confusing for me. Because this sentence describes what would happen if we do not use any of the options. So I guess either it needs to be above the options as a general description of the issue or I would keep it here and just remove the _In this situation_ because it sounds like this is what happens when you go OPtion 1.

Reply:
I moved above.